addons/l10n_sa/doc/cla/corporate/dvit.md

```diff
@@ -0,0 +1,15 @@
+Egypt, 2015-07-20
+
+DVIT.me agrees to the terms of the Odoo Corporate Contributor License
+Agreement v1.0.
+
+I declare that I am authorized and able to make this agreement and sign this
+declaration.
+
+Signed,
+
+Mohamed M. Hagag m.hagag@dvit.me https://github.com/mohamedhagag
```

Comment:
you need to sign it with the same email as the one you are [using to commit](https://help.github.com/articles/setting-your-email-in-git/) (`mohamedhagag1981...` in your case)